bitcoin/src/psbt/mod.rs

```diff
@@ -2249,7 +2249,7 @@ mod tests {
         };
         assert_eq!(
             t.fee().expect("fee calculation"),
-            prev_output_val - (output_0_val + output_1_val)
+            (prev_output_val - (output_0_val + output_1_val).unwrap()).unwrap()
```

Comment:
In 4d6911b12c0dec91ebf2ba1065224300e6231ad5:

You should only need one unwrap here.

Reply:
Ha, correct. Thanks